HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -1129,21 +1135,23 @@ template <typename T>
 void FastTimerService::printEvent(T& out, ResourcesPerJob const& data) const {
   printHeader(out, "Event");
   printEventHeader(out, "Modules");
-  auto const& source_d = callgraph_.source();
-  auto const& source = data.modules[source_d.id()];
-  printEventLine(out, source.total, source_d.moduleLabel());
+  printEventLine(out, data.source, "source");
```

Comment:
can you keep it as
```suggestion
  printEventLine(out, data.source, source_d.moduleLabel());
```
just in case it is ever different?